src/components/Request/RequestEntry.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Badge from 'react-bootstrap/Badge';
+import Button from 'react-bootstrap/Button';
+import axios from 'axios';
+
+
+const RequestEntry = (props) => {
+    const request = props?.request
+
+    const handleAccept = async (e) => {
+        e.preventDefault();
+        const payload = {
+            "fromUid" : request?.fromUid,
+            "toUid" : request?.toUid,
+            "requestid" : request?.transactionId
+        }
+        try {
+            axios.put(`http://127.0.0.1:5000/request/accept`, payload, {headers: {
```

Comment:
Don't hard-code IP and port here because we may forget to change them later. Import it from `../utils` or `../CLIENT`